freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -159,6 +130,17 @@ public void userSetPassword(String user, String password) throws FreeIpaClientEx
         return invoke("group_add_member", flags, params, Object.class);
     }
 
+    public Set<Group> groupFindAll() throws FreeIpaClientException {
+        List<String> flags = List.of();
+        Map<String, Object> params = Map.of(
```

Comment:
For consistency with userFindAll(), I expected the "all" param to be set here as well. BTW, when all is set to true, the return value includes the members of each group.

Reply:
good tip. that may come in useful as I refine the behavior.